doc/source/docs/howto_build_docs.rst

```diff
@@ -48,13 +48,19 @@ with::
 
 Now you are ready to generate the docs, so write::
 
+    cd doc
     make html
 
 in the ``doc/`` directory. If all goes well, this will generate a
 ``build/html`` subdirectory containing the built documentation. If you get
 a message about ``installed numpy != current repo git version``, you must
 either override the check by setting ``GITVER`` or re-install NumPy.
 
+Note that if you have built numpy into a virtual enviornment and getting an error that says ``numpy not found, cannot build documentation without...`` you need to override the makefile ``PYTHON`` variable at the commandline, so instead of writing ``make html`` write::
```

Comment:
The lines need to be < 80 characters in length. You need hard line breaks.

Reply:
I only just realised what I was doing wrong, I guess? Is it okay now? Thanks for walking me through though.